test/e2e/framework/ingress_utils.go

```diff
@@ -366,9 +366,10 @@ func CleanupGCEIngressController(gceController *GCEIngressController) {
 		By(fmt.Sprintf("WARNING: possibly leaked static IP: %v\n", ipErr))
 	}
 
-	// Fail if the controller didn't cleanup
+	// Logging that the GLBC failed to cleanup GCE resources on ingress deletion
+	// See kubernetes/ingress #431
```

Comment:
Removed, but it's not linking: https://github.com/nicksardo/kubernetes/blob/3e85c0f758fd31d7e39611694cd1d58424739374/test/e2e/framework/ingress_utils.go I'm not sure if that feature is markdown specific.  Would you prefer if I paste the whole link?